compiler/src/dotty/tools/dotc/core/PatternTypeConstrainer.scala

```diff
@@ -194,7 +214,7 @@ trait PatternTypeConstrainer { self: TypeComparer =>
    *  case classes without also appropriately extending the relevant case class
    *  (see `RefChecks#checkCaseClassInheritanceInvariant`).
    */
```

Comment:
The documentation comment above still talks about skolem wrapping, presumably that needs to be updated with the new scheme?